coursier.bzl

```diff
@@ -301,6 +316,12 @@ def _add_outdated_files(repository_ctx, artifacts, repositories):
         executable = True,
     )
 
+def _is_failing_build_if_inputs_change(repository_ctx):
```

Comment:
nit: should this method just be called `_fail_if_repin_required` ?

Reply:
Done.